### DISCUSSION TITLE
single-nuclei RNA-seq arrows in opposite direction

### DISCUSSION THREAD

**Discussion by balthasar0810** [theislab/scvelo#459] [Q&A]
Hi!

Following up on a few issues mentioning this (#118 #267), I do also see velocity arrows in the opposite direction (of what is known) for my single-nuclei data. As few people mentioned it, but I didn't find an explanation, where does this come from?

Best, Chris

**Comment by WeilerP** [comment]
@balthasar0810, can you share some figures, e.g. UMAP embedding and ideally the phase portraits of the top likelihood genes? This issue usually boils down to non-informative phase portraits (see e.g. #216).
